From c5433028cdbd59b5bccc1b4c82a582d83b1b0ce7 Mon Sep 17 00:00:00 2001
From: tpaviot <tpaviot@gmail.com>
Date: Thu, 12 Jan 2012 18:19:12 +0100
Subject: [PATCH] Fix syntax error in OSD_FileNode.cxx

Fix contributed by Massimo Del Fedele. Original post:
"""
2012/01/11 14:38
In OSD_FileNode.cxx, from line 17 :

"""
---
 src/OSD/OSD_FileNode.cxx |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/OSD/OSD_FileNode.cxx b/src/OSD/OSD_FileNode.cxx
index 7e3845c..74128d0 100644
--- a/src/OSD/OSD_FileNode.cxx
+++ b/src/OSD/OSD_FileNode.cxx
@@ -16,7 +16,7 @@
 
 #if!defined(TM_IN_SYS_TIME) && defined(HAVE_TIME_H)
 # include <time.h>
-#elif HAVE_SYS_TIME_H
+#elif defined(HAVE_SYS_TIME_H)
 # include <sys/time.h>
 #endif
 
-- 
1.7.9.1

